dvol_python/test_dvol.py

```diff
@@ -132,11 +155,15 @@ def test_commit_volume(self):
         self.assertTrue(commit.exists())
         self.assertTrue(commit.child("file.txt").exists())
         self.assertEqual(commit.child("file.txt").getContent(), "hello!")
+    if TEST_DVOL_BINARY:
+        test_commit_volume.todo = "not expected to work in go version"
 
     def test_list_empty_volumes(self):
         dvol = VoluminousOptions()
         dvol.parseOptions(ARGS + ["-p", self.tmpdir.path, "list"])
         self.assertEqual(dvol.voluminous.getOutput(), ["  VOLUME   BRANCH   CONTAINERS "])
+    if TEST_DVOL_BINARY:
```

Comment:
unittest's skipIf decorator seems a cleaner way to do this, not sure if trial has an equivalent decorator for TODO: